docs/how_to_write_packages.md

```diff
@@ -1,3 +1,265 @@
+
+In order to ease keeping track of those changes, Preflight packages have a version tag. That version is specified with the `package-version` property in the _policy manifest_.
+
+### The minimal _policy manifest_
+
+Let's just write the minimal _policy manifest_ possible.
+
+First, create a directory for this new package. We are going to create this new package under the `examples.jetstack.io` namespace, and we are going to name it `podsbestpractices`.
+
+Then create the `policy-manifest.yaml` file. The following fields are mandatory:
+
+- `schema-version`: indicates which schema is being used for the _policy manifest_. For the moment, there is only version `1.0.0`.
+
+- `namespace`, `id`, and `package-version`: these properties identify the package. `namespace` must be a FQDN and it is encouraged that `package-version` uses semver.
+
+Then, you should also declare the _data-gatherers_ that your rules are going to need. For this example, let's just use `k8s/pods`.
+
+Finally, it's time to declare the rules for the policy. Rules are organized into sections. Every section has an ID, a name, and a description. Also, every rule has its own ID, name, and description. Additionally, rules can have other metadata like a remediation advice or a set of related links.
+
+For simplicity's sake, this example contains just one section with one rule.
+
+```
+# preflight-packages/examples.jetstack.io/podsbestpractices/policy-manifest.yaml
+
+schema-version: "1.0.0"
+id: "podsbestpractices"
+namespace: "examples.jetstack.io"
+package-version: "1.0.0"
+root-query: "data.pods" # the concept of `root-query` is explained later in this doc
+data-gatherers:
+- k8s/pods
+sections:
+  - id: images
+    name: Images
+    description: "Restrictions over the images."
+    rules:
+      - id: tag_not_latest
+        name: "Tag is not latest"
+        description: >
+          Avoid using "latest" as tag for the image since.
+        remediation: >
+          Change your manifest and edit the Pod template so the image is pinned to a certain tag.
+        links:
+          - "https://kubernetes.io/docs/concepts/containers/images/"
+```
+
+## Writing the policy definition in REGO
+
+In the previous section, we created the _policy manifest_, which contains a human readable description of the rules in our policy. Now it's time to define the same rules in a language that is machine readable.
+
+### The REGO package
+
+Preflight relies on Open Policy Agent as the policy engine. REGO is OPA's language to define policies. You can find a comprenhensive [documentation](https://www.openpolicyagent.org/docs/latest/policy-language/).
+
+You can have multiple REGO files inside the directory of a Preflight package.  All the REGO rules corresponding to the _policy manifest_ rules must be in the same REGO package, and that package must be indicated in the _policy manifest_ using the `root-query` property.
```

Comment:
```suggestion
You can have multiple Rego files inside the directory of a Preflight package.  All the Rego rules corresponding to the _policy manifest_ rules must be in the same Rego package, and that package must be indicated in the _policy manifest_ using the `root-query` property.
```

Just basing this off how it's referred to here: https://www.openpolicyagent.org/docs/latest/policy-language/